buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -7,7 +9,7 @@ plugins {
 group = "io.opentelemetry.contrib"
 version = "1.3.0-alpha"
 
-base.archivesBaseName = "${rootProject.name}-${project.name}"
+base.archivesBaseName = "opentelemetry-${project.name}"
```

Comment:
Results in `io.java.contrib:opentelemetry-jmx-metrics` and `io.java.contrib:opentelemetry-aws-xray` which seem good to me